doc/en/skipping.rst

```diff
@@ -234,11 +234,11 @@ expect a test to fail:
     def test_function():
         ...
 
-This test will be run but no traceback will be reported
-when it fails. Instead terminal reporting will list it in the
-"expected to fail" (``XFAIL``) or "unexpectedly passing" (``XPASS``) sections.
+This test will run but no traceback will be reported when it fails. Instead terminal
+reporting will list it in the "expected to fail" (``XFAIL``) or "unexpectedly
+passing" (``XPASS``) sections.
 
-Alternatively, you can also mark a test as ``XFAIL`` from within a test or setup function
+Alternatively, you can mark tests as ``XFAIL`` from within a test or setup function
```

Comment:
I think the previous phrasing was slightly better, because the new one might imply that you can skip test A from within test B.

I would phrase it like this though:

``
 Alternatively, you can also mark a test as ``XFAIL`` from within the test or its setup function
imperatively:
```

Reply:
Updated PR based on feedback